server/version.js

```diff
@@ -0,0 +1,25 @@
+const superagent = require("superagent");
+const semver = require("semver");
+
+/**
+ * Verify the running API version is compatible with the UI
+ *
+ * @func
+ * @param {string[]} minVersion - the minimum API version required
+ * @param {string} apiUrl - the base url for the API
+ * @returns {N/A}
+ */
+async function verifyAPIVersion(minVersion, apiUrl) {
+    const response = await superagent.get(apiUrl).then(res => res.body);
```

Comment:
I think you should load the `minimumApiVersion` here instead of `run.js`.

Reply:
Yeah that's fair. My thought process was towards loading all options in the main file, but as of now that is the only option passed via file like that. Switched loading it in `version.js`